Mathlib/Tactic/NormCast/Tactic.lean

```diff
@@ -64,11 +64,11 @@ is rewritten to:            op (↑(↑(x : α) : β) : γ) (↑(y : β) : γ)
 when (↑(↑(x : α) : β) : γ) = (↑(x : α) : γ) can be proven with a squash lemma
 -/
 def splittingProcedure (expr : Expr) : MetaM Simp.Result := do
-  let Expr.app (Expr.app op x ..) y .. := expr | return {expr}
+  let Expr.app (Expr.app op x ..) y .. := expr | return {expr := expr}
```

Comment:
We really need to figure out something to disambiguate the set and record notations.